backend/game/cards/discard_cards/victory_point_manipulation_cards/heal.py

```diff
@@ -0,0 +1,10 @@
+from game.cards.discard_card import DiscardCard
+from game.cards.card import Card
+
+
+class Heal(DiscardCard):
```

Comment:
We are going to have to be very careful with this card and the turn action heal and making sure we are not overloading or misreferencing things but we can figure that out at a later date.

Reply:
I think this was actually written by @otaiwo70 and I accidentally pulled it in. Tagged so he sees your comment.